### DISCUSSION TITLE
Setting X_train using set_config function doesn't work

### DISCUSSION THREAD

**Discussion by Luna0298** [pycaret/pycaret#3552] [Q&A]
Hello, I'm trying to change X_train through the "set_config" function in a classification problem, but I'm getting the following error:
`Variable X_train not found or is not writeable. Possible writeable variables are: ['data', 'USI', 'fix_imbalance', 'exp_name_log', 'exp_id', 'html_param', 'n_jobs_param', 'fold_generator', 'memory', 'seed', 'fold_shuffle_param', 'idx', 'log_plots_param', 'fold_groups_param', 'target_param', 'logging_param', 'pipeline', 'gpu_param']`
Is this variable really not changeable manually? I saw that it could be changed in newer versions of pycaret (I'm using pycaret 3.0.1)

**Comment by ngupta23** [comment]
Does not look like this is pycaret 3.0.1. Maybe you are somehow using an older version. Please check.

**Reply by Luna0298** [reply]
![image](https://github.com/pycaret/pycaret/assets/51924525/aa330d75-a119-4bbc-88c5-ebd30bf2987f)
It is pycaret 3.0.1, i'm sure. 
I also tried with pycaret 3.0.0, it didn't work.

**Reply by ngupta23** [reply]
Please open an issue and provide a reproducible example (data + code) that we can copy and paste to reproduce the error. It will help us debug further.

**Reply by Luna0298** [reply]
Thanks. I opened an issue containing code reproducing the error, here is the link: https://github.com/pycaret/pycaret/issues/3555
